### DISCUSSION TITLE
Dark Mode UI?

### DISCUSSION THREAD

**Discussion by MichelKerkmeester** [webstudio-is/webstudio-community#172] [Q&A]
Hi Guys,

Any ETA on a Dark Mode version of Web Studio?
Saw some old posts about a workaround but all the links have been taken offline?

**Comment by kof** [comment]
can't be given any ETA, none of the most important features on the roadmap will justify spending time on it right now, so most likely somewhere 2025 https://github.com/webstudio-is/webstudio/issues/3818
